### ISSUE TITLE
from django.db.models.loading import get_model  Django 1.9+

### ISSUE THREAD

**Issue by smbasha47** [lehins/django-wepay#5]
I am trying to integrate wepay in Django 1.9+ i am getting bellow issue can you fix this ASAP

```
$ python manage.py wepay_app_create
Traceback (most recent call last):
  File "manage.py", line 27, in <module>
    execute_from_command_line(sys.argv)
  File "d:\Python27\lib\site-packages\django\core\management\__init__.py", line 353, in execute_from_command_line
    utility.execute()
  File "d:\Python27\lib\site-packages\django\core\management\__init__.py", line 327, in execute
    django.setup()
  File "d:\Python27\lib\site-packages\django\__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "d:\Python27\lib\site-packages\django\apps\registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "d:\Python27\lib\site-packages\django\apps\config.py", line 202, in import_models
    self.models_module = import_module(models_module_name)
  File "d:\Python27\lib\importlib\__init__.py", line 37, in import_module
    __import__(name)
  File "d:\Python27\lib\site-packages\djwepay\models.py", line 18, in <module>
    from djwepay.api import *
  File "d:\Python27\lib\site-packages\djwepay\api.py", line 4, in <module>
    from django.db.models.loading import get_model
ImportError: No module named loading
```

Thanks in Advance  
Basha

**Comment by smbasha47** [comment]
diff --git a/djwepay/api.py b/djwepay/api.py
index 714f771..61ede66 100644
--- a/djwepay/api.py
+++ b/djwepay/api.py
@@ -1,7 +1,14 @@
 from django.conf import settings
 from django.core.cache import cache
 from django.core.urlresolvers import reverse
-from django.db.models.loading import get_model
+try:
+    from django.db.models.loading import get_model
+    AppConfig = None
+    apps = None
+except:
+    from django.apps.config import AppConfig
+    from django.apps.registry import apps
+
 from django.utils.functional import LazyObject, curry
 
 from djwepay.signals import state_changed
@@ -56,7 +63,12 @@ def get_wepay_model(obj_name):
     model_name = get_wepay_model_name(obj_name)
     if model_name is None:
         return None
-    return get_model(*model_name.split('.'))
+    
+    if AppConfig is not None:
+        app_config = AppConfig(app.__name__, app)
+        return app_config.get_model(*model_name.split('.'))
+        
+    return get_model(*model_name.split('.')) 
 
 
 def is_abstract(obj_name):

**Comment by lehins** [comment]
My apologies, but I am way too busy with other projects to be able to maintain this one at the moment. If you would like to submit a pull request that integrates it with Django 1.9+ I'll be happy to review and merge it.

**Comment by lehins** [comment]
Fixed in 80bdbba91a3f6544cddfc409fd266dbae6508ed1
